### PR TITLE
set version of select2 in blueprints

### PR DESCRIPTION
**Pull request by jondayton**
with the release of select2 4.0.0, there are quite a few breaking changes

### CODE REVIEW

File: blueprints/ember-select-2/index.js [hunk 1]
```diff
@@ -2,6 +2,6 @@ module.exports = {
   normalizeEntityName: function() {},
 
   afterInstall: function() {
-    return this.addBowerPackageToProject('select2');
+    return this.addBowerPackageToProject('select2', '3.5.1');
```
Comment (kevin-brown) [inline blueprints/ember-select-2/index.js]: Shouldn't this be 3.5.2? It was the last release in 3.5.x.

**Comment by mattmcmanus** [comment]
+1

**Comment by Keeo** [comment]
+1

**Comment by iStefo** [comment]
You are right, this component is not ready for 4.0 (yet?). However, 3.5.2 is indeed the most recent version on 3.x, so I will close this PR but immediately set the version number myself. Thanks!

**Pull request closed without merging**